grpc/src/client/subchannel.rs

```diff
@@ -345,30 +353,34 @@ impl InternalSubchannel {
         let transport = self.transport.clone();
         let address = self.address().address;
         let state_machine_tx = self.state_machine_event_sender.clone();
-        let connect_task = tokio::task::spawn(async move {
```

Comment:
Do we have some kind of `vet` equivalent to ensure that task spawning (and other features provided by the runtime) are always only used from the runtime and not from other places (like tokio or the standard library)?